doc/customization_basic.rst

```diff
@@ -0,0 +1,150 @@
+- config.pypsa-earth.yaml.
+When you run the tool, a new file will be created in the folder: config.yaml.
+This is the file that contains all the specifications that will be executed in your simulation,
+and it is therefore the file that you must take care to edit correctly.
+
+If this is the first time you have approached these matters, you may be wondering why such a complex structure, the reason being that the config.yaml file that is generated is not a file that is tracked 
+differently from the other two. This makes it possible to work in such a way that the config.yaml file can be modified by the user whenever he wants to run a simulation, 
+while the config.distirbution.yaml and config.pypsa-earth.yaml files are only modified when the user wants to propose structural changes to the code.
+
+
+In the next steps we will see the basic settings in order to customize the configuration file, in case you are interested in more in-depth customization, you can refer to the configuration section in the Pypsa-Earth documentation.
+
+Choose the type of simulation:
+--------------------------------------
+It is possible to run simulations with the Pypsa Distribution tool in two modes:
+
+- tutorial mode: this is a faster simulation in which less data is used. It was created to allow users to become familiar with the tool, without necessarily having to run heavy simulations.
+  Remember: For realistic case studies it will be necessary to run the tool in full configuration and not in tutorial one.
+- full ( non-tutorial) mode: in this case a full simulation is performed and the use of committed resources is greater.
+
+Since the tutorial mode is designed only to become familiar with the tool, and not to perform analysis in a broad sense, it is possible to run the simulation in tutorial mode located in these nations: 
+
+- Nigeria [NG]
+- Benin [BJ]
+- Botswana [BW]
+- Morocco [MA]
+
+The tutorial mode is selectable by placing within the config file:
+
+.. code:: yaml
+
+    tutorial: true
+
+To perform a full simulation it will be necessary to set:
+
+.. code:: yaml
+
+    tutorial: false
+
+
+Specify the country of interest:
+--------------------------------------
+
+Selection of the country of interest it's possible using the "coutries" argument at the top of the config file.
+
+.. code:: yaml
+
+    countries: ["NG"]
+
+
+Please remember,before proceeding, that as has already been mentioned, in case you are running a simulation in tutorial mode you will only be able to select one nation from NG,BJ,BW or MA; 
+and that for realistic case studies it will be necessary to run the tool in full configuration
+
+Configure `coordinates`
+--------------------------
+Currently, the tool allows us to select the coverage area of interest by going to enter the coordinates of the vertices of the rectangular surface we want to be covered.
+At the end of the config.yaml file you will find the microgrid list section where you can go to enter the coordinates. 
```

Comment:
mention the `microgrig_list` parameter using the ` character so that it will be highlighted in the text